### PR TITLE
Banner: link to candidate-list and election posts

### PR DESCRIPTION
**Pull request by chalin**
- Following through from #1863
- Also, "normalized" titles and paths of earlier 2022 GC posts, to match what we did in 2021; and, we don't need the year in the file name since each post is in a folder for a given year.

Preview:

- Homepage for the banner: https://deploy-preview-1864--opentelemetry.netlify.app
- Posts redirect tests (for the two that were renamed):
  - https://deploy-preview-1864--opentelemetry.netlify.app/blog/2022/gc-elections-2022
  - https://deploy-preview-1864--opentelemetry.netlify.app/blog/2022/gc-elections-2022-list-of-candidates

/cc @jpkrohling 

---

### Screenshot

> image

### CODE REVIEW

**Review by jpkrohling — APPROVED** [review]

**Comment by chalin** [comment]
@jpkrohling - can you confirm that you saw the post path changes (see the opening comment for details).

Oh, and I need to fix one of the aliases, which is why I put this in draft mode.

**Comment by chalin** [comment]
Alias fixed. Ready for final review.

**Review by cartermp — APPROVED** [review]

**Comment by chalin** [comment]
Do you want to wait for @jpkrohling's feedback or merge this now @cartermp? I'll let you merge when you think it's time to do so.

File: content/en/blog/2022/gc-candidates.md
Comment (jpkrohling) [inline content/en/blog/2022/gc-candidates.md:6]: I tried this prior to this PR being merged and it works.

File: content/en/blog/2022/gc-elections.md
Comment (jpkrohling) [inline content/en/blog/2022/gc-elections.md:6]: I tried this prior to this PR being merged and it works.

**Review by jpkrohling — APPROVED** [review]
I just confirmed that the aliases work so that current links to those pages will continue working after this PR gets merged.

**Comment by svrnm** [comment]
then let's merge :-)

**Pull request merged**
